environment.yml

```diff
@@ -16,3 +16,4 @@ dependencies:
   - sqlalchemy
   - antlr-python-runtime==4.9.2
   - pyodbc
+  - mock
```

Comment:
Do we need this even if we use `unittest.mock`?

Reply:
No, I might have added it by accident.